src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/KeyWrapTests.cs

```diff
@@ -0,0 +1,908 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Encryption.Aes.Tests
+{
+    using Aes = System.Security.Cryptography.Aes;
```

Comment:
`using` in a namespace is not something we typically use. Intentional? Or take it out of the namespace?

Reply:
It has to be defined inside the block because it's overwriting the segment from the test namespace.  (Unless we feel like not sharing a namespace with the current AES tests).

The alternative is to give it a longer alias outside, or write `Cryptography.Aes` any time the type would come up on the inside.

Reply:
Ah, no, that's fine then.